src/objects/zcl_abapgit_object_ddlx.clas.abap

```diff
@@ -110,6 +110,16 @@ CLASS ZCL_ABAPGIT_OBJECT_DDLX IMPLEMENTATION.
   ENDMETHOD.
 
 
+  METHOD get_tadir_interface.
+    IF mo_tadir_interface IS NOT BOUND.
+      CREATE OBJECT mo_tadir_interface TYPE lcl_tadir
+        EXPORTING
+          io_outer = me.
+    ENDIF.
+    ri_tadir = mo_tadir_interface.
+  ENDMETHOD.
```

Comment:
this method `get_tadir_interface` seems like a workaround? suggest leaving this out and opening a new issue regarding dependency injection for `tadir_insert`

Reply:
done